modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -135,3 +135,32 @@ grains:
       - name.one
       - name.two
 ----
+
+
+
+== SSL Errors on Registration
+
+Some clients experience SSL errors when you register them in the {productname} {webui} using a bootstrap script.
```

Comment:
Same comments as before on the other pages. Here we should also have the list of affected OS.